src/components/MyAccount/IconListElement.tsx

```diff
@@ -0,0 +1,45 @@
+import { Icon } from "@nypl/design-system-react-components"
+import type { IconNames } from "@nypl/design-system-react-components"
+
+import styles from "../../../styles/components/MyAccount.module.scss"
+
+interface IconListElementPropType {
+  icon: IconNames
+  term: string
+  description: string
+}
+
+const IconListElement = ({
+  icon,
+  term,
+  description,
+}: IconListElementPropType) => {
+  return (
+    <>
+      <dt className={styles.iconDt}>
+        <Icon size="large" name={icon} title={`account profile ${term} icon`} />
```

Comment:
@EdwinGuzman I wasn't quite sure what the title of the icon should be indicating. Does this look okay?

Reply:
It does look okay but, for example, when `term` is "Email:", it'll render as `account profile Email: icon`. The colon can potentially make it confusing for screen readers.
The other part of these `Icon`s is that they have `aria-hidden="true"` by default so they _should_ be hidden to screen readers. This is also okay because they are decorative and the text immediately after describes it.

For now, try to remove the colon and follow up with Clare.

Reply:
removed the colon!

Reply:
Checked with Clare. She confirmed what you were saying - the icon title is redundant and won't be displayed anyway.